packages/main/src/DatePicker.ts

```diff
@@ -321,6 +321,15 @@ class DatePicker extends DateComponentBase implements IFormElement {
 	@property({ type: Boolean })
 	hideWeekNumbers!: boolean;
 
+	/**
+	 * Defines the open | closed state of the popover.
```

Comment:
On a second look, I would propose:
Defines the open or closed state of the popover.